build/unix/prepare-install-package.sh

```diff
@@ -117,13 +117,13 @@ for folder in ${folders[@]}; do
     cp "$installAssets/runtimeconfig.json" "$bundlePath/StardewModdingAPI.runtimeconfig.json"
 
     # installer DLL config
-    if [ $folder == "windows" ]; then
+    if [ "$folder" == "windows" ]; then
         cp "$installAssets/windows-exe-config.xml" "$packagePath/internal/windows/install.exe.config"
     fi
 
     # bundle root files
     for name in "StardewModdingAPI" "StardewModdingAPI.dll" "StardewModdingAPI.pdb" "StardewModdingAPI.xml" "steam_appid.txt"; do
-        if [ $name == "StardewModdingAPI" ] && [ $folder == "windows" ]; then
+        if [ $name == "StardewModdingAPI" ] && [ "$folder" == "windows" ]; then
```

Comment:
Shouldn't `$name` be quoted too while we're at it?
```suggestion
        if [ "$name" == "StardewModdingAPI" ] && [ "$folder" == "windows" ]; then
```